### ISSUE TITLE
the minimize option should be removed from the doc

### ISSUE THREAD

**Issue by j1348** [pingyuanChen/webpack-uglify-js-plugin#5]
it does not seem to be supported now.
I mean it should always be true.

**Comment by NCal** [comment]
I agree with J1348. 'Minimize' seems to do nothing when changed to false. The bundle is still minified.

**Comment by sjcalamia** [comment]
Indeed, I don't always want it to be minified, but that's not supported now. Debating removing the plugin entirely, or perhaps putting a larger conditional on the whole include.
